src/components/client/GaugesPage.tsx

```diff
@@ -29,7 +29,9 @@ const Gauges: NextPage = () => {
       >
         <PageHeader
           header="IQ Gauges"
+          hasBody={true}
           body="Track all gauges within our IQ platform."
+          hasExternalLink={false}
```

Comment:
You don't have to maintain ``hasExternalLink`` and ``hasBody`` again, since you are passing the ``externalLink`` and body as props.